lib/src/main/java/com/raqun/oyster/card/CardUtil.kt

```diff
@@ -0,0 +1,24 @@
+package com.raqun.oyster.card
+
+class CardUtil {
+
+    companion object {
+        fun getType(cardNumber: String): CardType {
```

Comment:
you can clean card number with the ext fun in CardExt.

Reply:
Updated